roles/kubernetes/node/templates/kubelet-container.j2

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
-/usr/bin/docker run --privileged --rm \
---net=host --pid=host --name=kubelet \
+/usr/bin/docker run --privileged \
+--net=host --pid=host --name=kubelet --restart=on-failure:5 \
```

Comment:
I'm not sure this changes anything as this container is under systemd service unit control, see https://github.com/kubernetes-incubator/kargo/blob/deea7bb87ba9f00f1afbce722feafc13a531d611/roles/kubernetes/node/templates/kubelet.service.j2#L27-L30